metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/lineage/Relationships.java

```diff
@@ -67,7 +68,7 @@ private RelatedEntitiesResult getRelatedEntities(String rawUrn, List<String> rel
     start = start == null ? 0 : start;
     count = count == null ? MAX_DOWNSTREAM_CNT : count;
 
-    return _graphService.findRelatedEntities("", newFilter("urn", rawUrn), "", QueryUtils.EMPTY_FILTER,
+    return _graphService.findRelatedEntities(ImmutableList.of(), newFilter("urn", rawUrn), ImmutableList.of(), QueryUtils.EMPTY_FILTER,
```

Comment:
It seems like its going to become confusing to provide either null or list... one other thing we couild do to simplify is just add another method that doesn't take either source types or dest types!

Reply:
addressed- made null all and empty list none